src/Skoruba.IdentityServer4.Admin.Api/Helpers/StartupHelpers.cs

```diff
@@ -205,5 +206,41 @@ public static void AddAuthorizationPolicies(this IServiceCollection services)
                         ));
             });
         }
+
+        public static void AddIdSHealthChecks<TConfigurationDbContext, TPersistedGrantDbContext, TIdentityDbContext, TLogDbContext, TAuditLoggingDbContext>(this IServiceCollection services, IConfiguration configuration, AdminApiConfiguration adminApiConfiguration)
+            where TConfigurationDbContext : DbContext, IConfigurationDbContext
+            where TPersistedGrantDbContext : DbContext, IPersistedGrantDbContext
+            where TIdentityDbContext : DbContext
+            where TLogDbContext : DbContext, IAdminLogDbContext
+            where TAuditLoggingDbContext : DbContext, IAuditLoggingDbContext<AuditLog>
+        {
+            var configurationDbConnectionString = configuration.GetConnectionString(ConfigurationConsts.ConfigurationDbConnectionStringKey);
+            var persistedGrantsDbConnectionString = configuration.GetConnectionString(ConfigurationConsts.PersistedGrantDbConnectionStringKey);
+            var identityDbConnectionString = configuration.GetConnectionString(ConfigurationConsts.IdentityDbConnectionStringKey);
+            var logDbConnectionString = configuration.GetConnectionString(ConfigurationConsts.AdminLogDbConnectionStringKey);
+            var auditLogDbConnectionString = configuration.GetConnectionString(ConfigurationConsts.AdminAuditLogDbConnectionStringKey);
+
+            var identityServerUri = adminApiConfiguration.IdentityServerBaseUrl;
+            services.AddHealthChecks()
+                .AddSqlServer(configurationDbConnectionString, name: "ConfigurationDb",
+                    healthQuery: "SELECT TOP 1 * FROM dbo.Clients")
```

Comment:
You were looking at the previous version, I already handled it in the latest commit: instead of hard coding the name of a table, I just pick the name of the first table in the db context, using the new helper method GetEntityTable.